src/interfaces/IQuestion.ts

```diff
@@ -1,11 +1,12 @@
 export default interface IQuestion {
   id?: number;
   title?: string;
-  email?: string;
+  userEmail?: string;
   question: string;
   answer?: string;
   grade: number;
   date?: string;
   course: number;
   theme: number;
+  anon?: boolean;
```

Comment:
Skal vi heller skrive det ut helt? `anonymous`

Reply:
Skrev det bare likt som det var gjort i backend for å være sikker på å ikke få noen feil når vi GETer fra question endpointet. Vet du om den bryr seg om hva de heter? Mener at det har litt å si, men kan godt hende jeg tar feil!